src/@orb.yml

```diff
@@ -1,6 +1,12 @@
 version: 2.1
 
-description: Build, preview and publish Leanpub books
+description: >
+  Build, preview and publish Leanpub books.
+  
+  You must obtain a Leanpub API key before running this orb.
```

Comment:
Please add a link to the documentation for how to get an API key: https://leanpub.com/help/api

Reply:
Done. :slightly_smiling_face: